app/components/ui/atoms/CalloutBox.js

```diff
@@ -23,6 +22,15 @@ const CloseButton = styled(CrossIcon)`
   width: ${th('space.3')};
 `
 
+const Frame = styled(props => <Box {...props} />).attrs({
```

Comment:
Note: none of this `Frame` code has changed. It's only been moved so that the CrossIcon and its stylings are next to each other